src/main/java/core/basesyntax/services/impl/AddOperationHandler.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.impl.StorageDaoImpl;
+import core.basesyntax.models.TransactionDto;
+import core.basesyntax.services.OperationHandler;
+
+public class AddOperationHandler implements OperationHandler {
+    private final StorageDao storageDao;
+
+    public AddOperationHandler() {
+        storageDao = new StorageDaoImpl();
```

Comment:
here you depend on the implementation
let's pass the implementation through the constructor